docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+version: '3.8'
+
+services:
+  web:
+    build: .
+    command: uvicorn src.main:app --host 0.0.0.0 --port 8000 --reload
```

Comment:
you execute that command two times, in dockerfile and here
as I wrote in a comment above, develop entrypoint.sh for that goal